tools/RAiDER/models/credentials.py

```diff
@@ -69,7 +69,7 @@ def _check_envs(model):
         key = os.getenv('RAIDER_ECMWF_ERA5_API_KEY')
         host = API_URLS['cdsapirc']
 
-    elif model in ('HRES'):
+    elif model in ('HRES', 'ERAI'):
```

Comment:
@jhkennedy dont think these should be sharing the same?

Reply:
@dbekaert I wasn't quite sure what to do with this one. 

For my [`setup_from_env`](https://github.com/dbekaert/RAiDER/blob/70a663d3a95c046cdbb31d7402b40306f22d0e9e/tools/RAiDER/models/credentials.py#L150-L152), I want to loop through [all supported models](https://github.com/dbekaert/RAiDER/blob/70a663d3a95c046cdbb31d7402b40306f22d0e9e/tools/RAiDER/models/credentials.py#L18-L24) and ensure their credentials are setup with the existing [`_check_envs`](https://github.com/dbekaert/RAiDER/blob/70a663d3a95c046cdbb31d7402b40306f22d0e9e/tools/RAiDER/models/credentials.py#L66) function.

`ERAI` is in the `API_FILENAME` dictionary and specifies that it uses the `ecmwfapirc` file (the same as `HRES`) but wasn't handled in `_check_envs` at all, and when passed to it, would just return `None` causing `check_api` to throw. 

Notably, `check_api` only ensures the config file exists, not that the correct credentials are in the config file.

So this change ensures all the config files are written and sufficient to pass testing. 

Really, if the crednetials for `ERAI` are different than `HRES`, the correct way to handle this would be to either:
1. drop support for `ERAI`, or
2. add an `ERAI` block to `_check_envs` to look for `RAIDER_ERAI_*` environment variables *and* update `check_api` to ensure the right credentials are in `ecmwfapirc`. This would mean appending credentials for `HRES`/`ERAI` if the other is present, changing the `update_flag` option to actually *update* (check + append) not *replace*, and probably a few other things.

Reply:
Since `ERAI` is not in the [allowed model's list](https://github.com/dbekaert/RAiDER/blob/dev/tools/RAiDER/models/allowed.py), and it sounds like `ERAI` has been moved from the ECMF API, I've dropped it from the credential checking module.

It could be added back in in a later PR that brings back `ERAI` support.